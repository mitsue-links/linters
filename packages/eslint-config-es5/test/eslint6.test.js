/* eslint strict: [2, "global"] */
'use strict';

const path = require('path');
const {lintFile} = require('./helper');

process.chdir(__dirname);

describe('ESLint 6.7.0', () => {
    test('6.7.0_ok.js', async () => {
        const filePath = '6.7.0_ok.js';
        const data = await lintFile(path.join('fixtures', 'eslint', filePath));

        expect(data).toStrictEqual([{
            filePath,
            messages: [],
        }]);
    });

    test('6.7.0_fail.js', async () => {
        const filePath = '6.7.0_fail.js';
        const data = await lintFile(path.join('fixtures', 'eslint', filePath));

        expect(data).toMatchSnapshot();
    });
});
