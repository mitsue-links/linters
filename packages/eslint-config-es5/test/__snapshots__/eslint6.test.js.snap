// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ESLint 6.7.0 6.7.0_fail.js 1`] = `
Array [
  Object {
    "filePath": "6.7.0_fail.js",
    "messages": Array [
      Object {
        "line": 6,
        "ruleId": "Parsing error: Unexpected token *",
        "severity": 2,
      },
    ],
  },
]
`;
